### PR TITLE
Update to use blocks-dev

### PR DESCRIPTION
**Pull request by Wattenberger**
Note to selves: we'll need to update the pre- & post-install scripts in githubnext/blocks to work with this repo structure

### CODE REVIEW

File: blocks.config.json [hunk 1]
```diff
@@ -0,0 +1,253 @@
+[
+  {
+    "type": "file",
+    "id": "code-block",
+    "title": "Code block",
+    "description": "A basic code block",
+    "sandbox": false,
+    "entry": "blocks/file-blocks/code/index.tsx",
+    "extensions": ["*"],
```
Comment (mattrothenberg) [inline blocks.config.json:9]: I'd love to move over to the `matches` API in earnest. Thoughts on using this major refactor as an opportunity to deprecate (or at least move away from) `extensions`?

File: blocks/file-blocks/css.tsx [hunk 2]
```diff
@@ -2,13 +2,13 @@ import { tw } from "twind";
...
 import { toJSON } from "cssjson";
-import { FileBlockProps } from "@githubnext/utils";
+import { FileBlockProps } from "@githubnext/blocks";
 
 export default function (props: FileBlockProps) {
   const { content } = props;
 
   const { tree, flattenedRules, widelyApplicableAttributes } = useMemo(() => {
-    console.log(content);
+    console.log("content", content);
```
Comment (mattrothenberg) [inline blocks/file-blocks/css.tsx:11]: drop?

File: blocks/folder-blocks/minimap/CircleText.tsx [hunk 3]
```diff
@@ -25,3 +24,8 @@ export const CircleText = ({ r = 10, rotate = 0, text = "", ...props }) => {
     </>
   );
 };
+
+let id = 0;
+function uniqueId(prefix = "") {
```
Comment (mattrothenberg) [inline blocks/folder-blocks/minimap/CircleText.tsx:29]: nice

File: blocks/folder-blocks/minimap/index.tsx [hunk 4]
```diff
@@ -10,6 +10,7 @@ export default function (props: FolderBlockProps) {
     const nestedTree = getNestedFileTree(tree)[0];
     return nestedTree;
   }, [tree]);
+  console.log(tree);
```
Comment (mattrothenberg) [inline blocks/folder-blocks/minimap/index.tsx:13]: drop

**Review by mattrothenberg — CHANGES_REQUESTED** [review]
A few cleanup nits. Thanks for getting this updated!

File: blocks.config.json (hunk 1 above)
Comment (jaked) [reply blocks.config.json:9]: agreed, we should take the opportunity to drop `extensions`

Comment (Wattenberger) [reply blocks.config.json]: awsm, dropped!

**Review by mattrothenberg — APPROVED** [review]

File: blocks/file-blocks/excalidraw/index.tsx [hunk 5]
```diff
@@ -16,6 +16,7 @@ if (typeof window !== "undefined") {
 export default function (props: FileBlockProps) {
   const { context, content, isEditable, onUpdateContent } = props;
   const [excalModule, setExcalModule] = useState<any>(null);
+  console.log("excalModule", excalModule);
```
Comment (jaked) [inline blocks/file-blocks/excalidraw/index.tsx:19]: really wish there was a :log: in PR comments

Comment (Wattenberger) [reply blocks/file-blocks/excalidraw/index.tsx]: doh!

**Pull request merged**